File: menu-service/src/main/java/com/food/delivery/menuservice/TestController.java
package com.food.delivery.menuservice;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
import reactor.core.publisher.Mono;

@RestController
public class TestController {

	@GetMapping("/menu/test")
	public Mono<String> test() {
		return Mono.just("HELLO FROM MENU SERVICE");
	}
}

File: menu-service/src/main/kotlin/com/food/delivery/menuservice/TestController.kt
package com.food.delivery.menuservice

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import reactor.core.publisher.Mono

@RestController
class TestController {

	@GetMapping("/menu/test")
	fun test(): Mono<String> {
		return Mono.just("HELLO FROM MENU SERVICE - KOTLIN")
	}
}
